test/input.txt

```diff
@@ -148,6 +148,16 @@
   clear
   list
 
+##########################################################
+# test sort command
+##########################################################
+  add Clark Kent p/12345678 e/clarkket@example.com
```

Comment:
You should add the address field (and any other **compulsory** fields) into the command too.